### ISSUE TITLE
Example of densenet usage

### ISSUE THREAD

**Issue by dfalbel** [rstudio/keras#138]
Is an example of densenet usage https://github.com/dfalbel/densenet is interesting for keras documentation?

**Comment by jjallaire** [comment]
Yes, I think that would be great! We could do it as a Keras example which
just links to the densenet package.

**Comment by dfalbel** [comment]
I have a small example here: https://github.com/dfalbel/densenet/blob/master/vignettes/cifar10-DenseNet-100-12.Rmd 

The best way is to submit a PR with this, like the other examples?

**Comment by jjallaire** [comment]
Yes, just submit a PR to the keras repo. Maybe include within the preamble
a link back to the package so users know where to install it.
